cmd/env.go

```diff
@@ -3,9 +3,13 @@ package cmd
 import (
 	"context"
 
+	logging "github.com/ipfs/go-log/v2"
+
 	"github.com/celestiaorg/celestia-node/node"
 )
 
+var log = logging.Logger("cmd")
```

Comment:
nit: Ideally, CLIs would write into stdout directly without logger or anything. Usually, when you run some cli util, it does not have fancy logging in it. Same could stay for the panic/fatal log. Super minor

Reply:
@Wondertan Can you open an issue for this then?

Reply:
https://github.com/celestiaorg/celestia-node/issues/1130